test/Microsoft.AspNetCore.Razor.OmniSharpPlugin.Test/OmniSharpTestBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Razor.Test.Common;
+
+namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin
+{
+    public class OmniSharpTestBase : TestBase
```

Comment:
Tried to group every bit of ugliness we have to endure from interoping between strong-named/non-strong named assemblies here.

Embrace the hack!